backend/api/Database/Models/Robot.cs

```diff
@@ -26,6 +26,8 @@ public class Robot
         [MaxLength(200)]
         public string SerialNumber { get; set; }
 
+        public string RobotAsset { get; set; }
```

Comment:
```suggestion
        public string CurrentAsset { get; set; }
```
Don't include the name of the parent object as part of the child parameter

Reply:
done